libs/auth-api-lib/src/lib/clients/models/client.model.ts

```diff
@@ -50,6 +53,16 @@ export class Client extends Model {
   })
   clientType!: string
 
+  @ForeignKey(() => Domain)
+  @Column({
+    type: DataType.STRING,
+    allowNull: true,
+  })
+  domainName?: string
+
+  @BelongsTo(() => Domain)
```

Comment:
It would be convenient for the IDS if this was also exposed in the API.

Reply:
Do you want to do that in a follow up PR when the IDS is updated as well?

Reply:
Ok.